src/cc3d/__init__.py

```diff
@@ -260,6 +382,19 @@ def connected_components_stack(
   or fully decompressing the array using arr.decompress()
   to obtain a numpy array (but presumably this will blow
   out your RAM since the image is so big).
+
+  Args:
+    stacked_images: A sequence of images to process.
+    connectivity: 
+      (2d) 6 [faces], 26 [faces+edges+corners]
+    return_N: Change return value to (CrackleArray, N).
+    out_dtype: The output dtype.
+    binary_image: Treat the input images as binary images.
+
+  Returns:
+    A CrackleArray containing the connected components
+    of the input images and optionally the number of
+    connected components in the image.
```

Comment:
On line 408, I found to silence mypy errors, I had to write this:

```
  # Assigned at beginning of loop
  # commented out to silence mypy 
  # change of type errors
  
  # bottom_cc_img = None 
  # bottom_cc_labels = None
```

Reply:
That fixed the last ones!

```
> mypy src/cc3d
Success: no issues found in 2 source files
```